### ISSUE TITLE
Проблем с приложение

### ISSUE THREAD

**Issue by rakovm** [kaloyan-raev/kindle-k3-l10n-bg#1]
Здравей,
Инсталирал съм приложението на едни kindle 3g и всичко си работеше чудесно, но сега когато се избере да се свали книга, тя се появява в списъка като номер с разширение .mobi и не иска да се отвори? Знаеш ли какво се случва?
Поздрави!

**Comment by kaloyan-raev** [comment]
Да. Проблемът се обсъжда тук: https://github.com/kaloyan-raev/chitanka4kindle/issues/12
